src/extension/background-script/actions/accounts/add.ts

```diff
@@ -24,12 +29,34 @@ const add = async (message: MessageAccountAdd) => {
     name,
   };
 
+  const mnemonic = await generateMnemonic({
+    action: "generateMnemonic",
+    origin: { internal: true },
+  });
+  setMnemonic({
+    args: {
+      id: accountId,
+      mnemonic: mnemonic.data,
+    },
+    action: "setMnemonic",
+    origin: { internal: true },
+  });
+
   state.setState({ accounts: tmpAccounts });
 
   if (!currentAccountId) {
     state.setState({ currentAccountId: accountId });
   }
 
+  edit({
```

Comment:
Why this separate edit call? Just to set the toggle to use the Mnemonic for LNURL-auth?

Reply:
yup needed to set lnurl-auth for mnemonic